app/scss/styles.scss

```diff
@@ -73,8 +73,9 @@ html {
     }
 
     p {
-      display: inline; // WHY!? im too afraid to change this...
+      display: block;
```

Comment:
have you checked that this hasn't affected anything else in the app?

Reply:
We checked and there doesn't appear to change anything else in the app.

Reply:
we've tested the app and it doesn't appear to make any difference to anything (apart from turning the instructions modal on questionPage.html from inline to block)..